### PR TITLE
Assignment4 code commit

### PR DESCRIPTION
**Pull request by unnikrishnanks-1**
Method which would extract and return the substring from a given Sentence provided.
Eg: ‘Corporation’ substring from name, String name =”Indian Oil Corporation Ltd”

### CODE REVIEW

File: src/test/java/com/training/sample/tests/Assignment4Test.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Assignment4Test {
+    Assignment4 assignment4 = new Assignment4();
+
+    @Test
+    public void testGetString() {
+        Assertions.assertEquals(
+                "Corporation", assignment4.getString("Indian Oil Corporation Ltd", 11, 22));
```
Comment (shrikanttechincal) [inline src/test/java/com/training/sample/tests/Assignment4Test.java:13]: endIndex should be 21

File: src/main/java/com/training/sample/pages/Assignment4.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.training.sample.pages;
+
+public class Assignment4 {
+    public String getString(String sentence, int startindex, int endindex) {
```
Comment (shrikanttechincal) [inline src/main/java/com/training/sample/pages/Assignment4.java]: variable should be in camel case Format

File: src/test/java/com/training/sample/tests/Assignment4Test.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.training.sample.pages.Assignment4;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Assignment4Test {
+    Assignment4 assignment4 = new Assignment4();
+
+    @Test
+    public void testGetString() {
```
Comment (PranavHanwante) [inline src/test/java/com/training/sample/tests/Assignment4Test.java:11]: change the name of testcase

File: src/test/java/com/training/sample/tests/Assignment4Test.java (hunk 1 above)
Comment (PranavHanwante) [inline src/test/java/com/training/sample/tests/Assignment4Test.java:13]: change the name of method

Comment (PranavHanwante) [reply src/test/java/com/training/sample/tests/Assignment4Test.java:13]: getValue

File: src/test/java/com/training/sample/tests/Assignment4Test.java (hunk 3 above)
Comment (PranavHanwante) [reply src/test/java/com/training/sample/tests/Assignment4Test.java:11]: testName

**Review by PranavHanwante — APPROVED** [review]

**Pull request merged**